core/trino-main/src/main/java/io/trino/server/TrinoSystemRequirements.java

```diff
@@ -93,7 +93,8 @@ else if ("Mac OS X".equals(osName)) {
 
     private static void verifyJavaVersion()
     {
-        Version required = Version.parse("11.0.15");
+        Version required = Version.parse("17+35");
```

Comment:
Why +35 here?

Reply:
That's the version we are using in testing

Reply:
Can we make this 17.0.3?

Reply:
Thats odd ... see https://docs.oracle.com/en/java/javase/17/install/version-string-format.html for standard .. so the more readable 17.0.3 should work

Reply:
Done.

Reply:
I think that this will fail now as docker-images are using 17.0.0. I'll update those to 17.0.3

Reply:
https://github.com/trinodb/docker-images/pull/134

Reply:
17.0.3 is required.